Nodejs/Product/Nodejs/Repl/SaveReplCommand.cs

```diff
@@ -25,12 +25,12 @@ public Task<ExecutionResult> Execute(IReplWindow window, string arguments)
         {
             if (string.IsNullOrWhiteSpace(arguments))
             {
-                window.WriteError("save requires a filename");
+                window.WriteError(Resources.ReplSaveNoFileName);
                 return ExecutionResult.Failed;
             }
             else if (arguments.IndexOfAny(Path.GetInvalidPathChars()) != -1)
             {
-                window.WriteError(string.Format(CultureInfo.CurrentCulture, "Invalid filename: {0}", arguments));
+                window.WriteError(string.Format(CultureInfo.CurrentCulture, Resources.ReplSaveInvalidFileName, arguments));
```

Comment:
>CurrentCulture [](start = 60, length = 14)

Not the `CurrentUICulture`?  Does it make a difference?

Reply:
Used throughout the solution, so not willing to risk that change right now